repo/packages/A/airfield/0/package.json

```diff
@@ -1,25 +1,26 @@
 {
-    "packagingVersion": "4.0", 
-    "upgradesFrom": [], 
-    "downgradesTo": [], 
-    "minDcosReleaseVersion": "1.10", 
-    "name": "airfield", 
-    "version": "0.1.0", 
-    "maintainer": "https://github.com/MaibornWolff/dcos-airfield", 
-    "description": "Airfield - easily start and manage zeppelin instances on DC/OS", 
-    "selected": false, 
-    "framework": false, 
-    "tags": [
-        "airfield", 
-        "zeppelin", 
-        "spark"
-    ], 
-    "postInstallNotes": "Airfield is being installed!\n\n\tDocumentation: https://github.com/MaibornWolff/dcos-airfield\n\tIssues: https://github.com/MaibornWolff/dcos-airfield", 
-    "postUninstallNotes": "Airfield is being uninstalled.",
-    "licenses": [
-        {
-            "name": "Apache License 2.0",
-            "url": "https://raw.githubusercontent.com/MaibornWolff/dcos-airfield/master/LICENSE"
-        }
-    ]
-}
+  "packagingVersion": "4.0",
+  "upgradesFrom": [],
+  "downgradesTo": [],
+  "minDcosReleaseVersion": "1.10",
+  "name": "airfield",
+  "version": "0.1.0",
+  "maintainer": "https://github.com/MaibornWolff/dcos-airfield",
+  "description": "Airfield - easily start and manage zeppelin instances on DC/OS",
+  "selected": false,
+  "framework": false,
+  "tags": [
+    "airfield",
+    "zeppelin",
+    "spark"
+  ],
+  "postInstallNotes": "Airfield is being installed!\n\n\tDocumentation: https://github.com/MaibornWolff/dcos-airfield\n\tIssues: https://github.com/MaibornWolff/dcos-airfield",
+  "postUninstallNotes": "Airfield is being uninstalled.",
+  "licenses": [
+    {
+      "name": "Apache License 2.0",
+      "url": "https://raw.githubusercontent.com/MaibornWolff/dcos-airfield/master/LICENSE"
+    }
+  ],
+  "lastUpdated": 1552664069000
+}
```

Comment:
(nit) can you ensure no newline characters are trimmed from files.

Reply:
feel free to add a newline at the end AND retain the newline if it is already present - applicable for ALL files in this PR.

Reply:
> Also can you do the same change for v5 as well. I _think_ mke and k8 cluster are the only v5 packages for now.
> Thanks :)

I think v5 packages should have been included in this change. I wrote a logic which would update all packages having version greater than v2.